domain/resourcegroup/runaway.go

```diff
@@ -126,11 +130,17 @@ func (rm *RunawayManager) markQuarantine(resourceGroupName, convict, watchType s
 		}
 		rm.queryLock.Unlock()
 	}
+	var endTime time.Time
+	if ttl < 0 {
+		endTime = longestTime
+	} else {
+		endTime = now.Add(ttl)
```

Comment:
As there is no restriction on the ttl value, what if this endTime exceeds the longestTime

Reply:
I think this situation is equivalent to unlimited